firebase-firestore/src/main/java/com/google/firebase/firestore/remote/BloomFilter.java

```diff
@@ -27,26 +27,25 @@ public class BloomFilter {
   private final int hashCount;
   private final MessageDigest md5HashMessageDigest;
 
-  public BloomFilter(@NonNull byte[] bitmap, int padding, int hashCount) {
+  public BloomFilter(@NonNull byte[] bitmap, int padding, int hashCount)
+      throws BloomFilterException {
```

Comment:
Remove `throws BloomFilterException` from the `BloomFilter` constructor. Now that `BloomFilterException` is a `RuntimeException` it does not need to be declared in a "throws" clause.